### PR TITLE
RHMAP-11059 Add default location for Geolocation if query fails

### PR DESCRIPTION
**Pull request by aidenkeating**
Currently when running the weather application in an emulator the
obtaining of a location will fail due to there being no location
provided by the emulated device unless it is set manually.

This change makes the user aware of this information as well as adding
a default value for the location inside the app logic so that even
if the obtaining of a real location fails the weather section of the
app can still be used.

This also includes a small restructuring of the functions to work
better with this updated flow.

### CODE REVIEW

**Comment by aidenkeating** [comment]
@secondsun Would you mind taking a look please? Does a version need bumped or anything?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Welcome Client
+§# Welcome Client
```
Comment (secondsun) [inline README.md]: Did you mean to add §?

Comment (aidenkeating) [inline README.md]: No, no I did not, thanks 😅

**Comment by jcesarmobile** [comment]
You should link the cordova.js in the index.html so it doesn't show the ugly permission message on iOS

**Comment by aidenkeating** [comment]
Thanks @jcesarmobile, I have added the script and the permissions problems appear to have gone when getting location on iOS. Could you take a look please?

**Comment by jcesarmobile** [comment]
I think I would put the "a default location has been set." after the "Please try again later." (https://github.com/feedhenry-templates/welcome-app/blob/master/www/js/app/views/viewbase.view.js#L15)

Also, I noticed that if I run `grunt browserify` the resulting www/main.js is different from the current one. It has nothing to do with your PR, but I think we should update the current www/main.js with the generated one.

**Comment by aidenkeating** [comment]
@jcesarmobile Added changes based on your feedback

**Comment by jcesarmobile** [comment]
I said after, not instead, but I think it looks better without the "Please try again later.", so LGTM

**Comment by aidenkeating** [comment]
Yeah I thought it looked a bit odd with both, I thought this looks better too. Does this need anything  else or is it good to merge?

**Pull request merged**